src/server/rest/v1/service/TagService.ts

```diff
@@ -775,9 +775,9 @@ export default class TagService {
         id: importedTag.id.toUpperCase(),
         visualID: importedTag.visualID,
         description: importedTag.description ? importedTag.description : `Badge ID '${importedTag.id}'`,
-        name: importedTag.name.toUpperCase(),
-        firstName: importedTag.firstName,
-        email: importedTag.email,
+        name: importedTag.name?.toUpperCase() || '',
```

Comment:
Maybe user props should be set only if there are a name, firstName and email present and all mandatory, so we don't have case where name is present and not firstName and we avoid to have also name: null.
What do you think?

Reply:
I agree, I made the modifications